test/cases/040_packages/032_bcc/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# SUMMARY: Check that the logwrite package works
+# LABELS:
+# REPEAT:
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+NAME=bcc
+
+clean_up() {
+	rm -rf ${NAME}-*
+}
+trap clean_up EXIT
+
+# Test code goes here
+linuxkit build -disable-content-trust -format kernel+initrd -name "${NAME}" test.yml
```

Comment:
please don't disable content trust for tests. All packages should be signed once this is ready to get merged